src/avram/model.cr

```diff
@@ -233,6 +233,8 @@ abstract class Avram::Model
       converter: JSONConverter({{ data_type }}),
       {% elsif data_type.id == Array(Float64).id %}
       converter: PG::NumericArrayFloatConverter,
+      {% elsif converter %}
```

Comment:
Originally I was going to use 

```crystal
{% elsif data_type.resolve < Enum %}
```

This would let us pick up the type and set the converter automatically; however, once I did this, it starts to resolve ALL the types. This means that in the `comment` model where we have `belongs_to post : Post`, the `Post` is undefined because it's resolved *before* crystal requires the `post.cr` file. It would end up being a breaking change since any model that references another later in alphabetical order would now have to be required in that file. (or you'd just require each model one by one in the proper order)